lib/plottr_components/src/components/tag/TagView.js

```diff
@@ -50,6 +50,52 @@ const TagViewConnector = (connector) => {
     deleteTag = (e) => {
       e.stopPropagation()
       this.props.actions.deleteTag(this.props.tag.id)
+      this.removeTags(this.props.tag.id)
+    }
+
+    removeTags = (tagId) => {
```

Comment:
This isn't the right place to remove those tags.
The DELETE_TAG action should be handled in the `cards`, `characters`, `notes` and `places` reducers, where we can remove tags from each of those pieces of state.
(In an ideal world, our components would have no logic in them except for highly dynamic/interesting UI display stuff).

Reply:
This is nearly it.  The next step is to only have one action to delete a tag.
That action can go into the tags actions file.
You only need to call that one action and the `DELETE_TAG` event with the tag's `id` will reach all the reducers.